packages/playground/src/components/branches-list.tsx

```diff
@@ -20,24 +35,50 @@ interface BranchProps {
   }
   pullRequest: {
     sha: string
-    status: string
+    // status: string
   }
 }
 
 interface PageProps {
   branches: BranchProps[]
 }
 
-export default function BranchesList({ branches }: PageProps) {
+const CopyButton = () => {
+  const [copied, setCopied] = useState(false)
+
+  useEffect(() => {
+    let timeoutId: number
+    if (copied) {
+      //add copy function here if we need in the future
+      timeoutId = window.setTimeout(() => setCopied(false), 2500)
+    }
+    return () => {
+      clearTimeout(timeoutId)
+    }
+  }, [copied])
+
+  return (
+    <Button variant="ghost" size="xs" onClick={() => setCopied(true)}>
+      <Icon
+        name={copied ? 'tick' : 'clone'}
+        size={16}
+        className={copied ? 'text-success ' : 'text-tertiary-background'}
+      />
+    </Button>
+  )
+}
+
+export const BranchesList = ({ branches }: PageProps) => {
   return (
     <Table variant="asStackedList">
       <TableHeader>
         <TableRow>
           <TableHead>Branch</TableHead>
           <TableHead>Updated</TableHead>
           <TableHead>Check status</TableHead>
-          <TableHead>Behind | Ahead</TableHead>
-          <TableHead>Pull request</TableHead>
+          <TableHead className="text-center">Behind | Ahead</TableHead>
```

Comment:
Let's hide this column till we have api integration. You need to use `/commits/calculate-divergence` api from gitness for this.

Reply:
hide it first and will fetch api in next PR